tests/unit/components/indicatorMenu/indicatorMenu.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { describe, expect, test, beforeEach, vi } from "vitest";
+import { render, screen } from "@testing-library/vue";
+import { userEvent } from "@testing-library/user-event";
+import { mockVuetify } from "../../mocks/mockVuetify";
+import { mockPinia } from "../../mocks/mockPinia";
+import router from "../../../../src/router";
+import IndicatorMenu from "../../../../src/components/indicatorMenu/IndicatorMenu.vue";
+
+const spyRouterPush = vi.spyOn(router, "push");
+
+describe("IndicatorMenu", () => {
+    let user;
+    const renderComponent = async () => {
+        await render(IndicatorMenu, {
+            global: {
+                plugins: [mockVuetify, mockPinia(), router]
+            }
+        });
+    };
+
+    const openMenu = async () => {
+        const button = await screen.findByText("Force of infection");
```

Comment:
Slightly more reliable would be getting by the class `indicator-menu-activator`, because other things might have the same text.